server/prisma/migrations/20231012180514_updated_ticketing_tables/migration.sql

```diff
@@ -0,0 +1,38 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `purchased` on the `eventtickets` table. All the data in the column will be lost.
+  - You are about to drop the column `eventticketid_fk` on the `singletickets` table. All the data in the column will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "orderitems" DROP CONSTRAINT "orderitems_orderid_fkey";
+
+-- DropForeignKey
+ALTER TABLE "singletickets" DROP CONSTRAINT "singletickets_eventticketid_fk_fkey";
+
+-- DropForeignKey
+ALTER TABLE "singletickets" DROP CONSTRAINT "singletickets_orderitemid_fk_fkey";
+
+-- AlterTable
+ALTER TABLE "contacts" ALTER COLUMN "seatingaccom" DROP DEFAULT,
+ALTER COLUMN "seatingaccom" SET DATA TYPE VARCHAR(255);
+
+-- AlterTable
+ALTER TABLE "eventtickets" DROP COLUMN "purchased",
+ADD COLUMN     "singleticket_fk" INTEGER;
+
+-- AlterTable
+ALTER TABLE "orders" ADD COLUMN     "checkout_sessions" VARCHAR(255);
+
+-- AlterTable
+ALTER TABLE "singletickets" DROP COLUMN "eventticketid_fk";
+
+-- AddForeignKey
+ALTER TABLE "eventtickets" ADD CONSTRAINT "eventtickets_singleticket_fk_fkey" FOREIGN KEY ("singleticket_fk") REFERENCES "singletickets"("singleticketid") ON DELETE SET NULL ON UPDATE NO ACTION;
+
+-- AddForeignKey
+ALTER TABLE "orderitems" ADD CONSTRAINT "orderitems_orderid_fkey" FOREIGN KEY ("orderid_fk") REFERENCES "orders"("orderid") ON DELETE CASCADE ON UPDATE NO ACTION;
+
+-- AddForeignKey
+ALTER TABLE "singletickets" ADD CONSTRAINT "singletickets_orderitemid_fk_fkey" FOREIGN KEY ("orderitemid_fk") REFERENCES "orderitems"("orderitemid") ON DELETE CASCADE ON UPDATE NO ACTION;
```

Comment:
My only concern is if any of this migration will have unintended effects on other parts of the app that we don't see. I know you pointed out the seed data being invalid which is okay b/c it can be fixed. Overall, it seems okay, just curious how you feel about it?

Reply:
The primary change I made was switching the ownership of the relationship between singltickets and eventickets, as far as I can tell no other code utilized the table so I think that should be fine. I also removed the purchased column from the eventickets table, for that I went through the repository and updated the locations where the field was used. Overall I think the changes should be good.

Reply:
Okay that sounds good! Just wanted to make sure b/c there are a lot of relationships in the DB!